src/app/courses-page/pipes/order-by.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from '@app/models';
+
+@Pipe({
+  name: 'orderBy'
+})
+export class OrderByPipe implements PipeTransform {
+  transform(arr: Course[]): Course[] {
+    return arr.sort((a, b) => b.creationDate.getTime() - a.creationDate.getTime());
```

Comment:
it's not obvious to get in which order it sorts
Usually, this kind of stuff is wrapped into a readable function
for example.
Don't forget to make the copy, `sort` mutates the original array
```typescript
export class OrderByPipe implements PipeTransform {
  transform(arr: Course[]): Course[] {
   const copy = [...arr];
    return copy.sort((a, b) => this.newestFirst(a, b));

Reply:
@SteveRusin Changed.
Thanks)